internal_docs/track_publishing/track_publish.md

```diff
@@ -0,0 +1,13 @@
+# Track publishing
```

Comment:
This is not visible after `mix docs`. Also, the diagram is linked incorrectly

Reply:
The diagram is still not visible
![image](https://user-images.githubusercontent.com/24651552/179728832-a412387c-319c-4605-b7d6-f4da5a9395ec.png)

What about renaming this document to `Engine Architecture` or `Architecture`?